.circleci/config.yml

```diff
@@ -1,27 +1,58 @@
 version: 2.1
 
+orbs:
+  docker: circleci/docker@1.6.0
+
 jobs:
-  build-docker-image:
+  quality-checks:
     docker:
-      - image: docker
+      - image: circleci/node:14
     steps:
-      - setup_remote_docker
       - checkout
-      - run: sh build.sh
+      - run:
+          name: Install Sensei
+          command: npm ci
+      - run:
+          name: Check code formatting
+          command: npm run prettier:check
+      - run:
+          name: Launch tests
+          command: npm test
+      - docker/dockerlint:
+          debug: true
 
-  quality-checks:
-    docker:
-      - image: circleci/node:14
+  build-docker-image:
+    executor: docker/docker
     steps:
+      - setup_remote_docker:
+          docker_layer_caching: true
       - checkout
-      - run: npm ci
-      - run: npm run prettier:check
-      - run: npm test
+      - docker/build:
+          image: zenika/sensei
+          tag: test
 
 workflows:
   version: 2.0
 
   build:
     jobs:
-      - build-docker-image
       - quality-checks
+      - build-docker-image
+      - docker/publish:
+          requires:
+            - build-docker-image
+          executor: docker/docker
+          use-remote-docker: yes
+          remote-docker-dlc: yes
+          update-description: yes
+          image: zenika/sensei
+          tag: test
+          docker-username: DOCKERHUB_USERNAME
+          docker-password: DOCKERHUB_PASS
+          deploy: yes
```

Comment:
This seems to be the default, could we leave it out?

Reply:
It was to be explicit.

Reply:
OK then :)